packages/InteractiveProfilingTool-Core.package/IPTStats.class/instance/noLeavesMessage.st

```diff
@@ -0,0 +1,4 @@
+constants
+noLeavesMessage
+
+	^ 'There are no leaves relevant to runtime performance.'
```

Comment:
Ich bin mir nicht sicher, ob ich richtig verstehe, wofür diese Message ist. Wenn ich das richtig sehe, wird sie ausgegeben, wenn ein Tally keine `ChildrenWithRelevantPerformanceImpact`, korrekt? Dann fände ich zum Beispiel 'There are no leaves with relevant runtime' passender, oder ganz konkret: 'There are no Leaves with runtime over specified relevance threshhold.' Wenn ich mich richtig erinnere, wird der Threshhold ja in der kommenden Iteration noch anpassbar.

Reply:
Wie gerade besprochen: Die Fehlermeldung ist irrelevant, weil der Fehlerfall niemals auftritt.